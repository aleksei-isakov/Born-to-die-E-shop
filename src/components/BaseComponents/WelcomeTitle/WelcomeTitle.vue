<template>
  <div class="welcome-title-banner">
    <h1 class="welcome-title">
      {{ title }}
    </h1>
  </div>
</template>

<script>
export default {
  name: 'WelcomeTitle',

  props: {
    title: [String, Number]
  }
};
</script>

<style scoped>
.welcome-title {
  color: #0000ff;
  font-size: 40px;
  font-weight: lighter;
}
.welcome-title-banner {
  display: inline-block;
  margin: 5px;
}
</style>
