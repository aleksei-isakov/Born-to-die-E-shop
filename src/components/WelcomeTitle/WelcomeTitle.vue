<template>
  <div class="welcome-title__banner">
    <h1 class="welcome-title">
      {{ title }}
    </h1>
  </div>
</template>

<script>
export default {
  name: 'WelcomeTitle',

  props: {
    title: {
      type: [String, Number],
      default: 'Welcome to Born2Die Market!',
      required: true
    }
  }
};
</script>

<style scoped>
.welcome-title {
  color: #0000ff;
  font-size: 40px;
  font-weight: lighter;
}
.welcome-title__banner {
  display: inline-block;
  margin: 5px;
}
</style>
