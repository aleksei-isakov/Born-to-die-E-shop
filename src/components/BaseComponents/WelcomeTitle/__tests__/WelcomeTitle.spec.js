import { createLocalVue, shallowMount } from '@vue/test-utils';
import WelcomeTitle from '../WelcomeTitle.vue';

describe('WelcomeTitle.vue', () => {
  let wrapper;
  const title = 'title';
  const localVue = createLocalVue();
  const renderWrapper = () => {
    wrapper = shallowMount(WelcomeTitle, {
      localVue,
      propsData: {
        title: title
      }
    });
  };

  beforeAll(() => {
    renderWrapper();
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
