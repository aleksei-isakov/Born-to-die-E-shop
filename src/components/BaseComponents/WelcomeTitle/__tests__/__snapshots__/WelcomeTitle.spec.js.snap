// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeTitle.vue should match snapshot 1`] = `
<div class="welcome-title-banner">
  <h1 class="welcome-title">
    title
  </h1>
</div>
`;
